Explorer/Assets/DCL/AvatarRendering/Wearables/Systems/ResolveWearableByPointerSystem.cs

```diff
@@ -204,11 +206,28 @@ private void FinalizeWearableDTO([Data] bool defaultWearablesResolved, in Entity
                         catch (AssertionException e) { ReportHub.LogError(new ReportData(GetReportCategory()), $"Cannot apply the DTO to the wearable {component.GetUrn()}: {e.Message}"); }
 
                         component.IsLoading = false;
+                        promise.LoadingIntention.Pointers.Remove(assetEntity.pointers[0]);
+                    }
+
+                    foreach (var currentPointer in promise.LoadingIntention.Pointers)
```

Comment:
The only reason that the pointer will remain in the list is because is not found in the `wearableCatalog`, right? But later, in `ReportDTOException` you request the wearable to the catalog again for each non-resolved pointer, which should end up in a cache miss thus an exception. Im missing something?

Reply:
This logic is now deprecated with the new list